tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Features/Login User Claims Validation.feature

```diff
@@ -131,3 +132,36 @@ Scenario: Get the claims of an authority user with invalid primary organisation
         | scope                | Organisation         |
         | organisation         | Manage               |
         | account              | Manage               |
+@5447
+Scenario: Get the relatedOrganisation claim of an Authority user with valid primary organisation id
+    Given Organisation Organisation 2 has a Parent Relationship to Organisation Organisation 1
+    And a user is logged in
+        | Username             | Password        | Scope        |
+        | PostmanPat@email.com | An0therPa$$w0rd | Organisation |
```

Comment:
I think it would be better to use either Penny or Snake here rather than Pat. While it's not really going to affect the outcome of the test, related organizations are only really relevant to buyers and I doubt the NHSD organization, which is the primary organization of an authority user, will ever have any related organizations.

(We could update the implementation to only add the `relatedOrganisationId` claims when the user is a buyer but I think it would be unnecessary based on the point above.)

However, from a purely semantic/pedantic point-of-view it would be better to use a buyer user here (and in the other tests you've added.)

Reply:
i put in a test for single and multiple claims for authority and a multiple claims for buyer, i'll update the authority multiple to be a buyer and delete the authority single one then